allauth/account/views.py

```diff
@@ -290,6 +292,19 @@ def get_context_data(self, **kwargs):
         )
         return ret
 
+    def get_initial(self):
+        intial = super(SignupView, self).get_initial()
+        email = self.request.GET.get("email")
+        if email:
+            try:
+                EmailValidator()(email)
```

Comment:
There is no need to spawn a validator yourself, just call this method:

https://docs.djangoproject.com/en/4.2/ref/validators/#validate-email